ansible/library/extract_log.py

```diff
@@ -91,7 +92,7 @@ def extract_line(directory, filename, target_string):
         file = open(path)
     result = None
     with file:
-        result = [(filename, line) for line in file if target_string in line]
+        result = [(filename, line) for line in file if target_string in line and 'nsible' not in line]
```

Comment:
nsible? not ansible?

Reply:
Yes. To match both 'Ansible' and 'ansible'.